### PR TITLE
Product details imageGallery Progress Made, main image styled

### PR DESCRIPTION
**Pull request by gordon1213**

### CODE REVIEW

File: client/src/components/product_details/ImageSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React,{useState,useEffect} from 'react';
+import Image from "./Image.jsx";
+const ImageSelect = (props) => {
+  console.log(props);
```
Comment (clee4037) [inline client/src/components/product_details/ImageSelect.jsx]: Please remove

**Review by clee4037 — COMMENTED** [review]
1 comment. Other than that, looks good!

File: client/src/containers/ProductDetails.jsx [hunk 2]
```diff
@@ -37,7 +37,19 @@ const ProductDetails = ({itemId}) => {
     fetchData();
   }, [itemId]);
   useEffect(()=> {
-    setStyle(styles[0]);
+    var def = false;
+    for(let x of styles)
+    {
+      if(x["default?"]=== true) {
```
Comment (LukeLarson2) [inline client/src/containers/ProductDetails.jsx:43]: Is this supposed to be styles[x]["default?"] as each key itself doesn't have a stored value inherently? Can you provide an example of the style object to clarify please?

Edit: noted it is a for...of loop so x stores the value at each key, would still like to validate that the key is labeled as "default?"

**Review by LukeLarson2 — APPROVED** [review]
Good to go, thank you for clarifications

**Pull request merged**